src/main/java/org/apache/mesos/scheduler/DefaultScheduler.java

```diff
@@ -124,6 +129,31 @@ private void initializeRecoveryScheduler() {
                 recoveryStatusRef);
     }
 
+    private void initializeApi() {
+        new Thread(new Runnable() {
+            @Override
+            public void run() {
+                JettyApiServer apiServer = new JettyApiServer(
```

Comment:
This is the DefaultScheduler, and so provides the default APIs.  Further, more flexible implementations of the Scheduler interface are possible.  This isn't one of them.  As you say setting up a Jetty Server is easy.  If service authors wish to server other elements they can do so in that way.